CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - Update Wazuh to version [4.6.0](https://github.com/wazuh/wazuh/blob/v4.6.0/CHANGELOG.md#v460)
+## Wazuh Docker v4.5.3
```

Comment:
Add an empty line between releases (between line 7 and 8).

Reply:
Done in https://github.com/wazuh/wazuh-docker/pull/983/commits/b18c06865039492e3e8c4e5ab1c3c8f0d87b404d